acceleration/fast_model_training_guide.md

```diff
@@ -248,3 +259,9 @@ In summary, with all the optimization, the training time of 8 V100 GPUs to achie
 ![brats benchmark](../figures/brats_benchmark.png)
 
 More details is available at [Brats distributed training tutorial](https://github.com/Project-MONAI/tutorials/blob/master/acceleration/distributed_training/brats_training_ddp.py).
+
+
+### 3. Pathology metastasis detection task
+
+- Algorithm experiments based on dataset analysis. (1) Training dataset too large to effectively utilize caching mechanisms. (2) Fast convergence within a few epochs, and simple network (Resnet18), such that there is little room for performance gain from training perspective. (3) The major bottleneck comes from data transformations, especially ColorJitter which is a random permutation of communication of four transforms.
+- We employed CuCIM IO for loading whole slide images and cuCIM transforms for performing the chain of transforms on GPU. In this way, we accelerating the data loading, data augmentation and preprocessing. We are getting 3.8X speedup on DGXA100. The chart below shows how it is compared to the baseline.
```

Comment:
Could you please help add the link to the Pathology tutorial?
And also add some charts to show more details.

Thanks.